src/Razor/Microsoft.AspNetCore.Razor.Language/src/Components/ComponentPreserveWhitespaceDirective.cs

```diff
@@ -13,7 +13,7 @@ internal static class ComponentPreserveWhitespaceDirective
             builder =>
             {
                 builder.AddBooleanToken(ComponentResources.PreserveWhitespaceDirective_BooleanToken_Name, ComponentResources.PreserveWhitespaceDirective_BooleanToken_Description);
```

Comment:
Isn't `@preservewhitespace false` the default now? Wondering if purely the existence of this directive attribute is sufficient.

Reply:
If you're asking whether we should allow `@preservewhitespace` on its own with no true/false value, then we discussed that when preservewhitespace was first implemented. At the time we agreed that since it's intended as a rare edge case, and people should nearly always leave the default, then it's not really advantageous to have a shorthand, plus we could always add one in a back-compatible way in the future.

Since we're in the last 1-2 days before ask mode, I'd recommend we stick with this. I don't think it's valuable enough to rush in a new syntax right now.

Reply:
Or if I'm misreading your suggestion could you please ping me ASAP to clarify? Thanks!